lib/build.js

```diff
@@ -338,6 +343,11 @@ export default {
     targetName = targetName || this.activeTarget[p];
 
     Promise.resolve(this.targets[p]).then(targets => {
+      if (atom.packages.isPackageActive('linter')) {
```

Comment:
Would it make sense to do this in the callback where all packages are activated (this already exists in `activate`) ?